galaxy_ng/app/access_control/access_policy.py

```diff
@@ -218,6 +218,8 @@ class CollectionAccessPolicy(AccessPolicyBase):
     NAME = 'CollectionViewSet'
 
     def can_update_collection(self, request, view, permission):
+        if not view.kwargs.get("path"):
+            return False
```

Comment:
Whats the scenario where this gets called on a CollectionViewSet without a path set?

The non /content/ case like `/api/automation-hub/v3/collections` should get a 'path' kwarg set to the default.

Reply:
on `/pulp/api/v3/` when generating the OpenAPI schema:
![image](https://user-images.githubusercontent.com/17153022/117173312-05972c80-ada3-11eb-8398-361273930fce.png)